CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## v5.0.0-alpha.24
+
+<!-- generated comparing v5.0.0-alpha.23..next -->
+
+_Jan 25, 2021_
+
+Big thanks to the 21 contributors who made this release possible: @eps1lon, @mbrookes, @mnajdova, @greguintow, @praveenkumar-kalidass, @oliviertassinari, @lukaselmer, @povilass, @eladmotola, @kutnickclose, @artola, @vinyldarkscratch, @KarimMokhtar, @mxsph, @tonysepia, @d3mp, @CyanoFresh, @xs9627, @vicasas, @TimonPllkrn, @duganbrett
+
+Here are some highlights:
+
+- Converted TODO components to emotion
+- Fixed various issues on API documentation pages
+
+TODO INSERT HIGHLIGHTS
+
+### `@material-ui/core@`
```

Comment:
```suggestion
### `@material-ui/core@v5.0.0-alpha.24`
```

Shuold we update the script to contain the version when this heading is package? Or at least add todo for us to update it

Reply:
Headings are not generated by the script. This has to be done manually.

Reply:
Might be interesting to validate in the future. But I'm not looking forward to parsing markdown so let's leave it at that for now.